test/annotation_utils.t

```diff
@@ -1894,4 +1873,4 @@ subtest 'annoutil_uniq_functions' => sub {
 
 RASTTestUtils::clean_up();
 
-done_testing();
+done_testing(38);
```

Comment:
YMMV, but I wouldn't bother with putting in this test count -- it is a nuisance to update it when you add more tests to the file.

Reply:
I thought it was a neat feature to track the test count.  You are right when adding or removing a test I would have to remember to update the count.  I will roll this back.